nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -542,7 +542,11 @@ public class InvokeHTTP extends AbstractProcessor {
 
     public static final Relationship RESPONSE = new Relationship.Builder()
             .name("Response")
-            .description("Response FlowFiles transferred when receiving HTTP responses with a status code between 200 and 299.")
+            .description("""
+            Response FlowFiles transferred when receiving HTTP responses with a status code between 200 and 299.
+            Enabling [Response Generation Required] changes routing behavior and sends FlowFiles to the Response relationship regardless of status code received.
+            """
+            )
```

Comment:
If the other relationships always show then their descriptions would need to reflect this conditional behavior as well.

For the others the conditional language might read
Enabling [Response Generation Required] changes routing behavior and nothing will route to this relationship.

Reply:
Recommend making this behavior extremely clear:

Response FlowFiles transferred when receiving HTTP responses with a status code between 200 and 299.

Enabling [Response Generation Required] changes routing behavior and while responses are routed to their proper relationships as defined a FlowFile is also always routed to the Response relationship as well regardless of status code received.

Reply:
Thanks for the additional recommendation, I agree that making it clearer is needed. How about this updated version:

Enabling [Response Generation Required] changes routing behavior, sending unsuccessful responses to their corresponding relationships and also sending FlowFiles to the Response relationship as well, regardless of status code received.

Reply:
+1